src/content/api/merchants.mdx

```diff
@@ -0,0 +1,627 @@
+        }
+      ]
+    }
+    ```
+  </CodePanel>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="PUT"
+  path="/api/merchants/{merchantId}"
+>
+  ## Update Merchant
+
+  This endpoint allows you to update a Merchant.
+
+  ### Attributes
+  <Properties>
+    <Property name="name" type="string">
+      Merchant name.
+    </Property>
+
+    <Property name="settlementConfig" type="object" experimental>
+      Merchant [Settlement Config](#settlement-config-model).
+    </Property>
+
+    <Property name="location" type="location" experimental>
+      Physical Location of Merchant. It is highly recommended that you provide this otherwise, users won't be able to find you with our [Merchant Search](#search-merchants) API if they perform a origin + distance query.
+    </Property>
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Error code="403" message="BANK_ACCOUNT_MISMATCH">
+      The bank account in the settlement config does not belong to the same account.
+    </Error>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="PUT" path="/api/merchants/{merchantId}">
+    ```bash
+    curl -X PUT https://service.centrapay.com/api/merchants/5ee0c486308f590260d9a07f \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "name": "Centrapay Café",
+        "location": {
+          "lat": "-36.8483579",
+          "lng": "174.7725834",
+          "city": "Auckland",
+          "postCode": "1010",
+          "country": "NZ"
+        },
+        "settlementConfig": {
+          "bankAccountId": "WRhAxxWpTKb5U7pXyxQjjY"
+        }
+      }'
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Response">
+    ```json
+    {
+      "id": "5ee0c486308f590260d9a07f",
+      "accountId": "yqwyya0rzz3vvshqw0474u89xtj5fn",
+      "name": "Centrapay Café",
+      "test": false,
+      "country": "NZ",
+      "onboardingStatus": "applied",
+      "location": {
+        "lat": "-36.8483579",
+        "lng": "174.7725834",
+        "city": "Auckland",
+        "postCode": "1010",
+        "country": "NZ"
+      },
+      "settlementConfig": {
+        "bankAccountId": "WRhAxxWpTKb5U7pXyxQjjY"
+      },
+      "createdAt": "2021-09-12T01:11:22.491Z",
+      "updatedAt": "2021-11-12T01:17:46.499Z",
+      "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+      "updatedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey"
+    }
+    ```
+  </CodePanel>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/merchants/{merchantId}/set-onboarding-status"
+>
+  ## Set Merchant Onboarding Status
+
+  This endpoint allows you to set the onboarding status of a Merchant.
+
+  ### Attributes
+  <Properties>
+    <Property name="onboardingStatus" type="string" required>
+      The onboarding status of the Merchant. See [Onboarding Statuses](#onboarding-statuses) for possible values.
+    </Property>
+
+    <Property name="onboardingStatusReason" type="string">
```

Comment:
@AndyClifford we could state here that an onboardingStatusReason is required if the status is `on-hold` or `deactivated`.